src/css/profile/wearable/changeable/theme-circle/checkboxradio.less

```diff
@@ -165,6 +165,8 @@ input[type="radio"] {
 /** CheckBox / Radio button on arc-listview **/
 .ui-arc-listview-carousel-item.ui-listview {
 	li {
+		padding-left: 30 * @px_base;
+		padding-right: 16 * @px_base;
```

Comment:
In wearable profile, we use the @unit_base by default.

Reply:
Recently we started using @px_base in Wearable profile as well in consistency with Mobile.

Reply:
Okay, I see.